resources/featherlight.scss

```
@@ -0,0 +1,91 @@
+@import "vars";
+
+// Based on Featherlight 1.7.14
+
+.featherlight {
+    display: none;
+    position: fixed;
+    top: 0; right: 0; bottom: 0; left: 0;
+    z-index: 2147483647;
```

```suggestion
    z-index: 1000;
```

Let's not go straight to the maximum. What if we wanted to display something else on top?